internal/resource/cuda-device.go

```diff
@@ -58,8 +58,8 @@ func (d *cudaDevice) GetDeviceHandleFromMigDeviceHandle() (Device, error) {
 	return nil, fmt.Errorf("GetDeviceHandleFromMigDeviceHandle is unsupported for CUDA devices")
 }
 
-// GetTotalMemoryMB returns the total memory for a device
-func (d *cudaDevice) GetTotalMemoryMB() (uint64, error) {
+// GetTotalMemoryMiB returns the total memory for a device
```

Comment:
QUestion: Do we want to add something that makes `MiB` clearer?

```suggestion
// GetTotalMemoryMiB returns the total memory for a device in mebibytes (2^20 bytes)
```